packages/arb-token-bridge-ui/src/components/syncers/PendingTransactionsUpdater.tsx

```diff
@@ -20,8 +20,13 @@ export function PendingTransactionsUpdater(): JSX.Element {
     app: { arbTokenBridge, arbTokenBridgeLoaded }
   } = useAppState()
   const { address } = useAccount()
+  const { chain } = useNetwork()
   const { resetTransfers } = useCctpState()
 
+  useEffect(() => {
+    resetTransfers()
+  }, [chain?.id, resetTransfers])
+
   useEffect(() => {
     resetTransfers()
   }, [address, resetTransfers])
```

Comment:
perhaps we can add `chain?.id` to this dep array instead?

Reply:
using chain id has an issue - when i changed my destination chain to a invalid CCTP network, the CCTP pending txs stay as empty divs:

<img width="1141" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/8a78870f-7da8-4815-81fe-234618facd2b">